async_simple/coro/test/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 file(GLOB coro_test_src "*.cpp")
 add_executable(async_simple_coro_test ${coro_test_src} ${PROJECT_SOURCE_DIR}/async_simple/test/dotest.cpp)
 
-target_link_libraries(async_simple_coro_test async_simple ${deplibs} ${testdeplibs})
+target_link_libraries(async_simple_coro_test ${deplibs} ${testdeplibs})
```

Comment:
Would it be harmful to remain it? If no, I prefer to remain it.

Reply:
if we keep it, we must move some source code to cpp file, feels not suitable

Reply:
Event.h not template class, we can move some implement to cpp